sys/shell/commands/sc_ccnl.c

```diff
@@ -170,10 +167,16 @@ int _ccnl_interest(int argc, char **argv)
         addr_len = gnrc_netif_addr_from_str(relay_addr, sizeof(relay_addr), argv[2]);
     }
 
+    memset(_int_buf, '\0', BUF_SIZE);
+    memset(_cont_buf, '\0', BUF_SIZE);
     for (int cnt = 0; cnt < CCNL_INTEREST_RETRIES; cnt++) {
         ccnl_send_interest(CCNL_SUITE_NDNTLV, argv[1], relay_addr, addr_len, NULL, _int_buf, BUF_SIZE);
-        ccnl_wait_for_chunk(_cont_buf, BUF_SIZE);
+        if (ccnl_wait_for_chunk(_cont_buf, BUF_SIZE) > 0) {
+            printf("Content received: %s\n", _cont_buf);
+            return 0;
+        }
```

Comment:
Pesronally, I would have put this error message into an `else` clause to bind it directly to the `ccnl_wait_for_chunk` result.
IOtherwise, i further error conditions get introduced in the future then this would require to touch more lines.